medusa/tv/series.py

```diff
@@ -1223,7 +1223,7 @@ def load_from_indexer(self, tvapi=None):
         self.classification = getattr(indexed_show, 'classification', 'Scripted')
         self.genre = getattr(indexed_show, 'genre', '')
         self.network = getattr(indexed_show, 'network', '')
-        self.runtime = int(getattr(indexed_show, 'runtime', 0))
+        self.runtime = int(getattr(indexed_show, 'runtime', 0) or 0)
```

Comment:
not fake. show has a runtime right? but its not there. Default show run time is 40 for drama and 20 for commedy. Don't see why is wrong